py/src/braintrust/logger.py

```diff
@@ -2446,16 +2462,14 @@ def log(
         :param metadata: (Optional) a dictionary with additional data about the test example, model outputs, or just about anything else that's relevant, that you can use to help find and analyze examples later. For example, you could log the `prompt`, example's `id`, or anything else that would be useful to slice/dice later. The values in `metadata` can be any JSON-serializable type, but its keys must be strings.
         :param metrics: (Optional) a dictionary of metrics to log. The following keys are populated automatically: "start", "end".
         :param id: (Optional) a unique identifier for the event. If you don't provide one, BrainTrust will generate one for you.
-        :param allow_log_concurrent_with_active_span: (Optional) in rare cases where you need to log at the top level separately from an active span on the logger, set this to True.
+        :param allow_concurrent_with_spans: (Optional) in rare cases where you need to log at the top level separately from using spans on the logger elsewhere, set this to True.
         """
-        if not allow_log_concurrent_with_active_span:
-            check_current_span = current_span()
-            if getattr(check_current_span, "parent_object", None) == self:
-                raise Exception(
-                    "Cannot run toplevel Logger.log method while there is an active span. To log to the span, use Span.log"
-                )
+        if self._called_start_span and not allow_concurrent_with_spans:
+            raise Exception(
+                "Cannot run toplevel `log` method while using spans. To log to the span, call `traced` and then log with `span.log`"
```

Comment:
isn't `experiment.start_span()` the more likely method?

Reply:
Idk don't most people start spans in python using `traced`? This will throw if they used `@traced` anywhere and then try to call `experiment.log`. This mirrors the example we use in the docs.

Reply:
Well I think if you're running an experiment, the pattern is something like:

```
for row in rows:
   with experiment.start_span() as span:
       ...
       output = task(input) # this is @traced
       ...
       span.log(input, output, ...)
```